code/datums/wounds/bones.dm

```diff
@@ -0,0 +1,282 @@
+	else
+		user.visible_message("<span class='danger'>[user] begins resetting [victim]'s [limb.name] with [I].</span>", "<span class='notice'>You begin resetting [victim]'s [limb.name] with [I]...</span>")
+
+	if(!do_after(user, base_treat_time * (user == victim ? 1.5 : 1), target = victim, extra_checks=CALLBACK(src, .proc/still_exists)))
+		return
+
+	if(victim == user)
+		limb.receive_damage(brute=15, wound_bonus=CANT_WOUND)
+		victim.visible_message("<span class='danger'>[user] finishes resetting [victim.p_their()] [limb.name]!</span>", "<span class='userdanger'>You reset your [limb.name]!</span>")
+	else
+		limb.receive_damage(brute=10, wound_bonus=CANT_WOUND)
+		user.visible_message("<span class='danger'>[user] finishes resetting [victim]'s [limb.name]!</span>", "<span class='nicegreen'>You finish resetting [victim]'s [limb.name]!</span>", victim)
+		to_chat(victim, "<span class='userdanger'>[user] resets your [limb.name]!</span>")
+
+	victim.emote("scream")
+	qdel(src)
+
+/*
+	Severe (Hairline Fracture)
+*/
+
+/datum/wound/brute/bone/severe
+	name = "Hairline Fracture"
+	desc = "Patient's bone has suffered a crack in the foundation, causing serious pain and reduced limb functionality."
+	treat_text = "Recommended light surgical application of bone gel, though splinting will prevent worsening situation."
+	examine_desc = "appears bruised and grotesquely swollen"
+
+	occur_text = "sprays chips of bone and develops a nasty looking bruise"
+	severity = WOUND_SEVERITY_SEVERE
+	interaction_efficiency_penalty = 2
+	limp_slowdown = 6
+	threshold_minimum = 60
+	threshold_penalty = 30
+	treatable_by = list(/obj/item/stack/sticky_tape, /obj/item/stack/medical/gauze)
+	status_effect_type = /datum/status_effect/wound/bone/severe
+	treat_priority = TRUE
+	scarring_descriptions = list("a faded, fist-sized bruise", "a vaguely triangular peel scar")
+
+
+/datum/wound/brute/bone/critical
+	name = "Compound Fracture"
+	desc = "Patient's bones have suffered multiple gruesome fractures, causing significant pain and near uselessness of limb."
+	treat_text = "Immediate binding of affected limb, followed by surgical intervention ASAP."
+	examine_desc = "has a cracked bone sticking out of it"
+	occur_text = "cracks apart, exposing broken bones to open air"
+	severity = WOUND_SEVERITY_CRITICAL
+	interaction_efficiency_penalty = 4
+	limp_slowdown = 9
+	sound_effect = 'sound/effects/crack2.ogg'
+	threshold_minimum = 115
+	threshold_penalty = 50
+	disabling = TRUE
+	treatable_by = list(/obj/item/stack/sticky_tape, /obj/item/stack/medical/gauze)
+	status_effect_type = /datum/status_effect/wound/bone/critical
+	treat_priority = TRUE
+	scarring_descriptions = list("a section of janky skin lines and badly healed scars", "a large patch of uneven skin tone", "a cluster of calluses")
+
+/datum/wound/brute/bone/treat(obj/item/I, mob/user)
```

Comment:
you should move this further up, so that it doesn't look like a compound fracture-only thing